### PR TITLE
Initial draft of SMS Overview page

### PR DESCRIPTION
**Pull request by marklewin**
## Description

An initial draft, but marking as "don't merge" because the building blocks are not in the new format yet. Completed this first because it will likely be translated first.

## Deploy Notes

By all means comment on the verbiage, but the page won't display the code in the "Getting Started" section until the building blocks are migrated to the new YAML format.

### CODE REVIEW

File: _documentation/messaging/sms/overview.md [hunk 1]
```diff
@@ -1,78 +1,76 @@
...
-* **[Number format](/voice/voice-api/guides/numbers)** - All phone numbers in the Nexmo APIs use E.164 format.
+* **[Number format](/voice/voice-api/guides/numbers)** - The SMS API requires phone numbers in E.164 format.
 
-* **[Authentication](/concepts/guides/authentication)** - Nexmo SMS API is authenticated with your account API Key and Secret.
+* **[Authentication](/concepts/guides/authentication)** - The SMS API authenticates using your account API key and secret.
 
-* **[Webhooks](/concepts/guides/webhooks)** - HTTP requests are made to your application web server so that you can act upon them. For example, the SMS API will send the delivery receipts and inbound SMS.
+* **[Webhooks](/concepts/guides/webhooks)** - The SMS API makes HTTP requests to your application web server so that you can act upon them. For example: inbound SMS and delivery receipts.
 
 ## Getting Started
```
[the remark this answers is not in the document]
Comment (marklewin) [reply _documentation/messaging/sms/overview.md]: Sorry @lornajane , I don't really understand what you mean!

File: _documentation/messaging/sms/overview.md [hunk 2]
```diff
@@ -1,78 +1,76 @@
 ---
 title: Overview
+navigation_weight: 1
+description: The SMS API overview.
 ---
 
-# Overview
+# SMS API Overview
```
[the remark this answers is not in the document]
Comment (marklewin) [reply _documentation/messaging/sms/overview.md]: Done.

File: _documentation/messaging/sms/overview.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lornajane) [reply _documentation/messaging/sms/overview.md]: I'd like the "Getting started" section to be first, so people see code samples really soon. And the rest can follow. I'm not a huge fan of the contents sections, but perhaps when everything else is so far down the page, they are still useful.

[the remark this answers is not in the document]
Comment (marklewin) [reply _documentation/messaging/sms/overview.md]: Have moved the Getting Started section, but renaming the page from Overview to SMS API puts it right at the bottom of the navigation menu. Must be hard-coded in NDP?

screen shot 2018-11-01 at 12 32 04

**Pull request merged**